docs/knife_commands.md

```diff
@@ -0,0 +1,92 @@
+Knife commands
+==============
+
+# Modifying a single box
```

Comment:
It's not really `Modifying a single box` - more `Creating/destroying a single box`.